src/app/public/modules/affix/affix.directive.ts

```diff
@@ -81,6 +105,18 @@ export class SkyAffixDirective implements OnChanges, OnDestroy {
   @Input()
   public affixVerticalAlignment: SkyAffixVerticalAlignment;
 
+  /**
+   * Fires when the affixed element's offset changes.
+   */
+  @Output()
+  public affixOffsetChange = new EventEmitter<SkyAffixOffsetChange>();
+
+  /**
+   * Fires when the affixed element's overflow container is scrolled.
+   */
+  @Output()
+  public affixOverflowScroll = new EventEmitter<void>();
```

Comment:
Should we complete `affixOverflowScroll` and `affixOffsetChange` ?

Reply:
Good catch.